### DISCUSSION TITLE
Problem of verificaition

### DISCUSSION THREAD

**Discussion by vidomicm** [community/community#52266] [GitHub Education]
Hello，
Does anyone know how to solve these two problems？
I don't know what should I edit in my profile to meet the requirements in the email 
无语死了，尝试了七八遍了，不论是用摄像头照相还是用上传照片都不能满足第一个要求。（网上说的上传材料和网站个人姓名都要用英文也尝试了，个人简介也加了，不知道到底哪里有问题）
![image](https://user-images.githubusercontent.com/115550146/230631560-0bfb8132-4b80-4426-b69e-990562ff9b9f.png)

**Comment by samus-aran** [comment]
Hi @vidomicm 👋 

We'd love to help! Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Comment by xdouworld** [comment]
hi @vidomicm 
1. You can fork some high star java or go projects and let the machine verify that you don't look like a robot.
2. Modify your profile to match the information you uploaded. For example, if you upload Xuexin.com, remember to translate into English. Wait a few minutes after the profile modification is complete. Verify again.
3. If that still fails, try to register a new account and register again.
1.你可以fork一些高星的java或者go项目，让机器验证你不像机器人。
2.修改一下你的个人profile，和你上传的信息里面一致。比如你上传的是学信网的话，记得要翻译成英文。修改profile完成等待几分钟。再次进行验证。
3.如果还是失败的话，尝试重新注册一个账号，再次注册。

**Comment by Yi-Baoxiang** [comment]
解决了吗兄弟，可以教教我吗，我用的是校园卡，我也是一直user profile的问题，关注你了，感谢。

**Comment by Yi-Baoxiang** [comment]
兄弟，我还有几个问题：
1.姓名用英文的还是中文的
2.bio里面具体写啥
3.手机拍照上传图片它说可信度不高，非得让我电脑设备拍照，但是我之前打印了一份英文成绩单电脑设备根本拍不清楚，你的意思是在手机上登陆Github去申请吗，谢谢了！

--------------原始邮件--------------
发件人："Zha&nbsp;Tong ***@***.***&gt;;
发送时间：2023年4月11日(星期二) 晚上10:18
收件人："community/community" ***@***.***&gt;;
抄送："易宝祥 ***@***.***&gt;;"Comment ***@***.***&gt;;
主题：Re: [community/community] Problem of verificaition (Discussion #52266)
-----------------------------------

 

 
我这个账号接收到邮件了，还在验证中，应该是人工验证了。我新注册了一个账号已经过了。这个东西很玄幻。我推荐你使用学信网的英文翻译版本。手机拍照进行验证

**Comment by samus-aran** [comment]
Hi @Yi-Baoxiang & @xdouworld thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. I will keep this open as @vidomicm is still seeking an answer at the moment but we'll lock/hide any posts in languages other than English in this discussion going forward .
